website/client/main.js

```diff
@@ -26,6 +29,9 @@ Vue.config.productionTip = IS_PRODUCTION;
 Vue.use(i18n);
 Vue.use(StoreModule);
 
+Vue.component('b-dropdown', BoostrapDropdown);
```

Comment:
Do you need to declare components like this? Shouldn't you just import to your .vue file and set the component property?